lib/reply.js

```diff
@@ -458,9 +458,11 @@ function onSendEnd (reply, payload) {
   if (payload === undefined || payload === null) {
     // according to https://tools.ietf.org/html/rfc7230#section-3.3.2
     // we cannot send a content-length for 304 and 204, and all status code
-    // < 200.
+    // < 200
+    // A sender MUST NOT send a Content-Length header field in any message
+    // that contains a Transfer-Encoding header field.
```

Comment:
```suggestion
    // that contains a Transfer-Encoding header field.
    // See: https://tools.ietf.org/html/rfc7230#section-3.3.2
```

Reply:
This url has been added at the first line of this code comments